openshift/openshift.go

```
@@ -417,3 +417,244 @@ func getSecretReferencesServiceAccount(u unstructured.Unstructured) []v1.ObjectR
+			if len(seLinuxOpts) == 0 {
+				delete(sc, "seLinuxOptions")
+			} else {
+				sc["seLinuxOptions"] = seLinuxOpts
+			}
+		}
+
+		// If securityContext is now empty, remove it entirely
+		if len(sc) == 0 {
+			unstructured.RemoveNestedField(modified.Object, scPath...)
+		} else {
+			unstructured.SetNestedMap(modified.Object, sc, scPath...)
```

golangci-lint (2.12.2)

[error] Error return value of `unstructured.SetNestedMap` is not checked

(errcheck)